docs/how-to/observability-with-prometheus-stack.md

```diff
@@ -0,0 +1,64 @@
+# Configure Observability with Prometheus Stack
+
+KubeAI provides a vLLM PodMonitor resource to scrape vLLM metrics.
+
+The podMonitor can be enabled by using the following KubeAI helm values:
+
+```yaml
+metrics:
+  prometheusOperator:
+    vLLMPodMonitor:
+      # Enable creation of PodMonitor resource that scrapes vLLM metrics endpoint.
+      enabled: true
+      labels: {}
+```
+
+If you want to manually create the PodMonitor please take a look at the KubeAI helm chart [vLLM PodMonitor template](https://github.com/substratusai/kubeai/blob/main/charts/kubeai/templates/vllm-pod-monitor.yaml).
+
+
+## Deploying Prometheus Operator
+The Prometheus Operator is a Kubernetes operator that manages Prometheus and its related components.
+The Prometheus Stack includes Grafana and Prometheus.
+
+Add Prometheus Helm repo:
+```sh
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+
+Install the Prometheus Stack and ensure PodMonitor without special labels works:
+```sh
+helm install prometheus prometheus-community/kube-prometheus-stack -f - <<EOF
+prometheus:
+  prometheusSpec:
+    podMonitorSelectorNilUsesHelmValues: false
+    ruleSelectorNilUsesHelmValues: false
+    serviceMonitorSelectorNilUsesHelmValues: false
+    probeSelectorNilUsesHelmValues: false
+EOF
+```
+
+## Enable KubeAI vLLM PodMonitor
+
+Install the KubeAI provided PodMonitor:
```

Comment:
I would prefer to move the values section for this from the intro to here.

Reply:
done